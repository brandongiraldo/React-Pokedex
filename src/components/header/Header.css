.header {
  background: #154DA4;
  color: white;
}
