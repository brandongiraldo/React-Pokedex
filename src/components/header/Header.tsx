import * as React from 'react';
import './Header.css';

interface HeaderProps {
  // no-op
}

const Header: React.SFC<HeaderProps> = props => {
  return (
    <div className={'container-fluid'}>
      <div className={'row'}>
        <div className={'header col-xs-12 col-sm-12 col-md-12 col-lg-12'}>{props.children}</div>
      </div>
    </div>
  );
};

export default Header;
