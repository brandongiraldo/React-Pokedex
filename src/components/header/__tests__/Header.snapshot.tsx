import * as React from 'react';
import {create} from 'react-test-renderer';
import Header from '../Header';

describe('Header Snapshots', () => {
  test('Simple Header with Text', () => {
    const component = create(<Header>{'Heres a Header with some text'}</Header>);
    expect(component).toMatchSnapshot();
  });
});
