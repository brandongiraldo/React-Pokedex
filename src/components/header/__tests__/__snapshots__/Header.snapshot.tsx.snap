// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshots Simple Header with Text 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="header col-xs-12 col-sm-12 col-md-12 col-lg-12"
    >
      Heres a Header with some text
    </div>
  </div>
</div>
`;
